src/interfaces/Order.ts

```diff
@@ -0,0 +1,15 @@
+export interface Order{
+    /*
+    * Notice the Order interface is NOT a one for one with the Order table in our api.
+    * This is due to Orders being slightly complex and actually broken into two different tables for normalization
+    * 1). We don't need an ID because Entity FW will handle that for us
+    * 2). We are adding a cart object here that will keep track of our list of products that will eventually be added to the OrderItem table in the DB
+    */
+    id: string;
```

Comment:
To keep consistent with ids in our api, should we just go ahead and switch our id definitions to numbers instead of strings?

Reply:
I think thats a good idea